content/en/ref/python/sdk/automations/_index.md

```diff
@@ -43,29 +38,6 @@ Automations in W&B (`wandb.automations`) provide event-driven workflow automatio
 | [`MetricThresholdFilter`](./metricthresholdfilter/) | Filter runs based on metric value comparisons against thresholds. |
 | [`MetricChangeFilter`](./metricchangefilter/) | Filter runs based on metric value changes over time. |
 
-
-## Common Use Cases
-
-### Model Performance Monitoring
-- Alert when model accuracy drops below a threshold
-- Notify team when training loss plateaus
-- Trigger retraining pipelines based on performance metrics
-
-### Artifact Management
-- Send notifications when new model versions are created
-- Trigger deployment workflows when artifacts are tagged
-- Automate downstream processing when datasets are updated
-
-### Experiment Tracking
-- Alert on failed or crashed runs
-- Notify when long-running experiments complete
-- Send daily summaries of experiment metrics
-
-### Integration Workflows
-- Update external tracking systems via webhooks
-- Sync model registry with deployment platforms
-- Trigger CI/CD pipelines based on W&B events
```

Comment:
@johndmulhausen Can we remove this and instead point to the Automation docs? It makes the landing pages longe (i.e. longer to get to code examples), and it's covered (or should be) in the dev guide itself.